react/features/deeplinking/components/DeeplinkingDesktopPage.js

```diff
@@ -0,0 +1,158 @@
+/* @flow */
```

Comment:
Do analytics have to be added to this page?

Reply:
Hmm. Good question.

Reply:
I'm working on adding some!